### ISSUE TITLE
2.0.6 npm package uncompiled?

### ISSUE THREAD

**Issue by mrosack** [HNeukermans/ng2-signalr#68]
This may be an issue in my project setup somewhere, but I can't use version 2.0.6 of the library because the NPM package isn't the compiled javascript version.  For example, in 2.0.6, `index.js` and the `bundles` directory don't exist in `node_modules/ng2-signalr`, and I get lots of compilation errors like:

```
ERROR in ./~/ng2-signalr/src/services/connection/i.signalr.connection.ts
Module parse failed: E:\<redacted>\node_modules\ng2-signalr\src\services\connection\i.signalr.connection.ts Unexpected token (6:7)
You may need an appropriate loader to handle this file type.
| import { ConnectionStatus } from "./connection.status";
| 
| export interface ISignalRConnection {
|     readonly status: Observable<ConnectionStatus>;
|     readonly errors: Observable<any>;
 @ ./~/ng2-signalr/src/ng2-signalr.ts 1:0-80
 @ ./~/ng2-signalr/index.ts
 @ ./ClientApp/app/app.module.shared.ts
 @ ./ClientApp/app/app.module.client.ts
 @ ./ClientApp/boot-client.ts
 @ multi event-source-polyfill webpack-hot-middleware/client?path=%2F__webpack_hmr ./ClientApp/boot-client.ts
```

In 2.0.3 everything works fine, and the compiled `index.js` exists in the NPM package.  Was it intentional to publish the package without the compiled javascript?

**Comment by HNeukermans** [comment]
will fix asap; I see my .npmignore file has gone...

**Comment by HNeukermans** [comment]
fixed in v2.1.0
